### ISSUE TITLE
Multiple GPUs error

### ISSUE THREAD

**Issue by decltypeme** [cmusatyalab/openface#298]
### Context of the issue.
I am trying to run the training script with ``-nGPU 2``. However, I get ``Some of weight/gradient/input tensors are located on different GPUs. Please move them to a single one.``

### Actual behavior.

Here, is the Stack Trace I get 

```
: ...e/ubuntu/torch/install/share/lua/5.1/threads/threads.lua:179: [thread 1 endcallback] ...e/ubuntu/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 1 callback] /home/ubuntu/torch/install/share/lua/5.1/nn/Container.lua:67: 
In 1 module of nn.Sequential:
/home/ubuntu/torch/install/share/lua/5.1/nn/THNN.lua:110: Assertion `THCTensor_(checkGPU)(state, 5, input, output, weight, columns, ones)' failed. Some of weight/gradient/input tensors are located on different GPUs. Please move them to a single one. at /tmp/luarocks_cunn-scm-1-7020/cunn/lib/THCUNN/generic/SpatialConvolutionMM.cu:70
stack traceback:
	[C]: in function 'v'
	/home/ubuntu/torch/install/share/lua/5.1/nn/THNN.lua:110: in function 'SpatialConvolutionMM_updateOutput'
	.../torch/install/share/lua/5.1/nn/SpatialConvolutionMM.lua:63: in function <.../torch/install/share/lua/5.1/nn/SpatialConvolutionMM.lua:53>
	[C]: in function 'xpcall'
	/home/ubuntu/torch/install/share/lua/5.1/nn/Container.lua:63: in function 'rethrowErrors'
	/home/ubuntu/torch/install/share/lua/5.1/nn/Sequential.lua:44: in function </home/ubuntu/torch/install/share/lua/5.1/nn/Sequential.lua:41>
	[C]: in function 'xpcall'
	...e/ubuntu/torch/install/share/lua/5.1/threads/threads.lua:234: in function 'callback'
	/home/ubuntu/torch/install/share/lua/5.1/threads/queue.lua:65: in function </home/ubuntu/torch/install/share/lua/5.1/threads/queue.lua:41>
	[C]: in function 'pcall'
	/home/ubuntu/torch/install/share/lua/5.1/threads/queue.lua:40: in function 'dojob'
	[string "  local Queue = require 'threads.queue'..."]:13: in main chunk
```

### OS and hardware information.

+ Operating system: Ubuntu 14.04.5 LTS
+ Torch version: 7
+ GPU type (if using): GRID K520

**Comment by decltypeme** [comment]
This has been figured out. It was missing to do something like this before running:
```
export CUDA_AVAILABLE_DEVICES=1,2
```
